lib/datadog/tracing/span.rb

```diff
@@ -60,7 +61,8 @@ def initialize(
         type: nil,
         trace_id: nil,
         service_entry: nil,
-        links: nil
+        links: nil,
+        events: nil
```

Comment:
Can the added parameters be documented in the docstring, specifically their expected types?